src/participant/prompts/query.ts

```diff
@@ -2,17 +2,21 @@ import * as vscode from 'vscode';
 
 import { getHistoryMessages } from './history';
 
+export const MAX_TOTAL_PROMPT_LENGTH = 10000;
```

Comment:
The number came from mms :)
https://github.com/10gen/mms/blob/master/server/src/main/com/xgen/cloud/naturallanguagequery/_public/svc/BuildQueryPromptUtils.java#L18C27-L18C50
But I will check if we can use maxInputTokens from the model, this will be more efficient if they continue to change the context window size.

Reply:
I keep MAX_TOTAL_PROMPT_LENGTH as a fallback for the first prompt, and then when maxInputTokens becomes available from the model I will replace it with the real number.

Reply:
Ah is `maxInputTokens` not available until we do a message?